### PR TITLE
Implement generateOrderUiStringAndLoc (sub_470b76)

### PR DESCRIPTION
**Pull request by duncanspumpkin**

### CODE REVIEW

File: src/OpenLoco/Windows/Vehicle.cpp [hunk 1]
```diff
@@ -2342,17 +2344,73 @@ namespace OpenLoco::Ui::Windows::Vehicle
             return Vehicles::OrderRingView(head->orderTableOffset);
         }
 
-        static std::pair<Map::Pos3, const char*> sub_470B76(const Vehicles::Order& order, uint8_t orderNum)
+        static std::pair<Map::Pos3, std::string> sub_470B76(const Vehicles::OrderRingView::Iterator& order, uint8_t orderNum)
         {
-            static loco_global<char[512], 0x0112CC04> _stringFormatBuffer;
+            std::stringstream ss;
+            ss << ControlCodes::inline_sprite_str;
+            // TODO: remove addition
```
Comment (LeftofZen) [inline src/OpenLoco/Windows/Vehicle.cpp:2351]: TODO

File: src/OpenLoco/Windows/Vehicle.cpp [hunk 2]
```diff
@@ -2342,17 +2344,73 @@ namespace OpenLoco::Ui::Windows::Vehicle
             return Vehicles::OrderRingView(head->orderTableOffset);
         }
 
-        static std::pair<Map::Pos3, const char*> sub_470B76(const Vehicles::Order& order, uint8_t orderNum)
+        static std::pair<Map::Pos3, std::string> sub_470B76(const Vehicles::OrderRingView::Iterator& order, uint8_t orderNum)
```
Comment (LeftofZen) [inline src/OpenLoco/Windows/Vehicle.cpp:2347]: method name?

**Review by LeftofZen — DISMISSED** [review]
remove todo comment, and add a name for this function

File: src/OpenLoco/Windows/Vehicle.cpp (hunk 1 above)
Comment (duncanspumpkin) [reply src/OpenLoco/Windows/Vehicle.cpp:2351]: i mean its still there. I could just go and address it though.

Comment (LeftofZen) [reply src/OpenLoco/Windows/Vehicle.cpp]: If you intend to leave the comment that's fine, but I guess does it make sense to anyone else? Will they know what you mean if they come across it?

**Pull request merged**